jobs/JOBSPROC_RAP_DUMP

```diff
@@ -152,51 +154,51 @@ export SCRIPTSobsproc=${SCRIPTSobsproc:\
 # ---------------------------------------
 # Paths for dump and bufr_dumplist items
 # ---------------------------------------
-export HOMEbufr_util=${HOMEbufr_util:\
--${NWROOT:?}/bufr_util.${bufr_util_ver:?\
-"###FATAL ERROR \"bufr_util_ver\" is not set"}}
+export HOMEbufr_dump=${HOMEbufr_dump:\
+-${PACKAGEROOT:?}/bufr_dump.${bufr_dump_ver:?\
+"###FATAL ERROR \"bufr_dump_ver\" is not set"}}
 
-export USHbufr_util=${USHbufr_util:-$HOMEbufr_util/ush}
-export FIXbufr_util=${FIXbufr_util:-$HOMEbufr_util/fix}
+export USHbufr_dump=${USHbufr_dump:-$HOMEbufr_dump/ush}
+export FIXbufr_dump=${FIXbufr_dump:-$HOMEbufr_dump/fix}
 
-# NOTE: path to dump executables defaults to $HOMEbufr_util/exec in dumpjb
+# NOTE: path to dump executables defaults to $HOMEbufr_dump/exec in dumpjb
 
 # directory path to data dump script executed in model script
 # -----------------------------------------------------------
-export ushscript_dump=${ushscript_dump:-$USHbufr_util}
+export ushscript_dump=${ushscript_dump:-$USHbufr_dump}
 
 # ---------------------------------------------------------
 # Paths for combined obsproc and dump items
 # ---------------------------------------------------------
 
 # directory paths to SSMI dump re-processing executed in bufr_dump_obs
 # --------------------------------------------------------------------
-export EXECPREP=${EXECPREP:-$HOMEbufr_util/exec}
+export EXECPREP=${EXECPREP:-$HOMEbufr_dump/exec}
 export PARMPREP=${PARMPREP:-$HOMEobsproc/parm}
-export FIXPREP=${FIXPREP:-$HOMEbufr_util/fix} # also path to expanded NAM
+export FIXPREP=${FIXPREP:-$HOMEbufr_dump/fix} # also path to expanded NAM
                                               #  domain mask file
 
 # directory paths to ERS, QuikSCAT & ASCAT dump re-processing executed in
 #  bufr_dump_obs
 # -----------------------------------------------------------------------
-export EXECWAVE=${EXECWAVE:-$HOMEbufr_util/exec}
+export EXECWAVE=${EXECWAVE:-$HOMEbufr_dump/exec}
 export PARMWAVE=${PARMWAVE:-$HOMEobsproc/parm}
-export FIXWAVE=${FIXWAVE:-$HOMEbufr_util/fix}
+export FIXWAVE=${FIXWAVE:-$HOMEbufr_dump/fix}
 
 # directory paths to SSMI, TRMM & WindSAT dump re-processing executed in
 #  bufr_dump_obs
 # ----------------------------------------------------------------------
-export EXECbufr=${EXECbufr:-$HOMEbufr_util/exec}
+export EXECbufr=${EXECbufr:-$HOMEbufr_dump/exec}
 export PARMbufr=${PARMbufr:-$HOMEobsproc/parm}
-export FIXbufr=${FIXbufr:-$HOMEbufr_util/fix}
+export FIXbufr=${FIXbufr:-$HOMEbufr_dump/fix}
 
 # ---------------------------
 # Paths for non-obsproc items
 # ---------------------------
 
 # TANK is root directory path to observational database used in dumpjb
 # --------------------------------------------------------------------
-export DCOMROOT=${DCOMROOT:-/gpfs/dell1/nco/ops/dcom}
+export DCOMROOT=${DCOMROOT:-/lfs/h1/ops/canned/dcom}
 export TANK=${TANK:-${DCOMROOT}/prod}
```

Comment:
Note, prod is now included in the $DCOMROOT definition. Please change to:
export TANK=${TANK:-${DCOMROOT}}